### ISSUE TITLE
Download badge rather than link to it?

### ISSUE THREAD

**Issue by OldhamMade** [ewjoachim/coverage-comment-action#25]
Hi, would it be possible for the badge from Shields.io to be downloaded and added to the wiki directly? 

It would reduce load on shields.io, since the badge would only be requested once from them per workflow execution, rather than each time the README is loaded. 

It would also mean the badge can be displayed for private repositories, since shields.io wouldn't need to call back to the (inaccessible) wiki.

**Comment by ewjoachim** [comment]
Yes we could do that. I believe that shields.io is supposed to handle this kind of load, but I understand your usecase.

**Comment by ewjoachim** [comment]
Just a note: we'll still need to store some info in the wiki, not just the file, because this is used to know the evolution of coverage rate, which is among the reported info.

**Comment by ewjoachim** [comment]
So I've made [python-coverage-comment-action](https://github.com/ewjoachim/python-coverage-comment-action) and I'm not sure I want to spend a lot of time maintaining this action too. If you're interested in taking over, let me know :)

**Comment by OldhamMade** [comment]
Thanks so much! I'll take a look.

Any specific reason why you moved this functionality over to another repo rather than including it in this one?

**Comment by ewjoachim** [comment]
I didn't move this functionality in particular. What happened is that:
- I needed to redo a huge part of the action in order to support external PRs that need a dedicated workflow
- Plus I saw that I was limited by the cobertura format and that it actually didn't bring "standardization" at all: it's a bad format and it doesn't really help

So I created a new version of this action that solves both issues and more:
- Supports comments on external PRs
- Uses the .coverage Python format which lets me handle combining multiple files
- Is written in testable code (not fully tested yet, though)

And I think I'm going to focus on developing that new action rather than this one because I don't think I'm going to be using this action in any of my projects.

(but to be clear, I haven't developed the feature mentionned in this issue #25 neither here or there)

Thus... You seem to be using this action, and its existing issues seems not to be bothering you too much. I'm offering to give you the ownership :)

**Comment by OldhamMade** [comment]
Ah, I see!

Actually, I'll probably look at using the other one since I need to store the SVG badge in the wiki for local access.

However, I don't use Python so the .coverage format probably isn't available from the tooling I have… I'll see whether I can raise a PR on the new project to include other formats.

I'll close this ticket since the other repo covers this feature. 👍🏻

**Comment by ewjoachim** [comment]
Nooo sorry I wasn't clear :( The other action **doesn't have** this feature either.
